.github/workflows/main.yml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install .[dev]
+	  python -m pip install .[dev]
+          python -m pip install pytest
```

Comment:
I believe `pip install .[dev]` already installs `pytest`, so this isn't really necessary.

Reply:
I don't think it did in my forked repo, but I could be mistaken